src/core_plugins/metrics/public/components/vis_types/table/config.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component, PropTypes } from 'react';
+import uuid from 'uuid';
+import DataFormatPicker from '../../data_format_picker';
+import createSelectHandler from '../../lib/create_select_handler';
+import createTextHandler from '../../lib/create_text_handler';
+import FieldSelect from '../../aggs/field_select';
+import Select from 'react-select';
+import YesNo from '../../yes_no';
+import ColorRules from '../../color_rules';
+
+class TableSeriesConfig extends Component {
+
+  componentWillMount() {
+    const { model } = this.props;
+    if (!model.color_rules || (model.color_rules && model.color_rules.length === 0)) {
+      this.props.onChange({
+        color_rules: [{ id: uuid.v1() }]
+      });
+    }
+  }
+
+  render() {
+    const defaults = { offset_time: '', value_template: '' };
+    const model = { ...defaults, ...this.props.model };
+    const handleSelectChange = createSelectHandler(this.props.onChange);
+    const handleTextChange = createTextHandler(this.props.onChange);
+
+    const functionOptions = [
+      { label: 'Sum', value: 'sum' },
+      { label: 'Max', value: 'max' },
+      { label: 'Min', value: 'min' },
+      { label: 'Avg', value: 'mean' },
+      { label: 'Overall Sum', value: 'overall_sum' },
+      { label: 'Overall Max', value: 'overall_max' },
+      { label: 'Overall Min', value: 'overall_min' },
+      { label: 'Overall Avg', value: 'overall_avg' },
+      { label: 'Cumlative Sum', value: 'cumlative_sum' },
+    ];
+
+    return (
+      <div>
+        <div className="vis_editor__series_config-container">
+          <div className="vis_editor__series_config-row">
+            <DataFormatPicker
+              onChange={handleSelectChange('formatter')}
+              value={model.formatter}
+            />
+            <div className="vis_editor__label">Template (eg.<code>{'{{value}}/s'}</code>)</div>
+            <input
```

Comment:
This input is missing a [correct label](https://github.com/elastic/kibana/blob/master/style_guides/accessibility_guide.md#label-every-form-element).